### ISSUE TITLE
Program Files folder cannot be written and javaw error

### ISSUE THREAD

**Issue by KarlosFanta** [mikeberger/borg_calendar#12]
Please add an installation manual or add some tips during setup.
borg_installer-1.8.0.1.jar installation on Win7 64bit:
This directory C:\Program Files\BORGCalendar can not be written! Please chose another directory"
Right-click Run As Admin did not show up, I had to do the following:
-> Copied installborg  .jar to c:\
-> Right click "Command prompt" in the start menu (just search for it) and select "Run as administrator", - c: + Enter  cd \ + Enter
-> "java -jar installborg.jar" + Enter
Now it works 100% :-)

when I do the borg_installer-1.8.0.1(1).exe
Error: Windows cannot find 'javaw'. make sure you typed the name correctly.... Error opening file
even if I run as Administrator.  (Win7 64 bit)  java Std edition V7 update71 build 1.7.0_71-b14 jdk

**Comment by mikeberger** [comment]
Someone else reported a similar problem. It was caused by Java 8 in that case. I have been unable to create a working .exe installer using IzPack. The .exe created by Izpack cannot find java 8.

I switched to use launch4j to create the installer .exe and it seems to work. The latest installer is here if you want to give it a try: https://github.com/mikeberger/borg_calendar/releases/tag/v1.8.1_beta3

I think that Launch4j is better at finding java than IzPack. So, even if this problem is not due to having java 8 on your machine, perhaps the new launcher will fix it.
